### DISCUSSION TITLE
Cant install

### DISCUSSION THREAD

**Discussion by dxur** [solidusio/solidus#5668] [Troubleshooting]
bin/rails g solidus:install

/usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require': cannot load such file -- json (LoadError)
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap/compile_cache/json.rb:49:in `init!'
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap/compile_cache/json.rb:42:in `install!'
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap/compile_cache.rb:34:in `setup'
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap.rb:68:in `setup'
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap.rb:112:in `default_setup'
	from /usr/lib/ruby/gems/3.0.0/gems/bootsnap-1.18.3/lib/bootsnap/setup.rb:5:in `<top (required)>'
	from /home/DxUr/Documents/Projects/eshop/shop/config/boot.rb:4:in `require'
	from /home/DxUr/Documents/Projects/eshop/shop/config/boot.rb:4:in `<top (required)>'
	from bin/rails:3:in `require_relative'
	from bin/rails:3:in `<main>'

**Comment by jarednorman** [comment]
Can you please provide more information?
